docs/templates/contributing_template.md

```diff
@@ -0,0 +1,281 @@
+of MIOpen and thereby, assumed that any deep learning kernels could be contributed 
+into MIOpen. In the same vein, they were unaware about the composable kernel library 
+and its potential for developing modular, performant kernels. 
+
+Below is a template for the external contributor's guidelines. 
+The template offers a starting point for each library owner which could be amended
+to suit the respective library. 
+
+The final guidelines of each library should be posted as CONTRIBUTING.md on GitHub repo or 
+could be posted on their landing page or the Wiki tab of GitHub project.
+
+Below is the template of contributor guidelines, followed by a sample for the MIOpen 
+library of ROCm. 
+--> 
+
+## Issue Discussion ##
+
+Please use the GitHub Issues tab to notify us of issues.
+
+* Use your best judgement for issue creation. If your issue is already listed, upvote the issue and
+  comment or post to provide additional details, such as how you reproduced this issue.
+* If you're not sure if your issue is the same, err on the side of caution and file your issue.
+  You can add a comment to include the issue number (and link) for the similar issue. If we evaluate
+  your issue as being the same as the existing issue, we'll close the duplicate.
+* If your issue doesn't exist, use the issue template to file a new issue.
+  * When filing an issue, be sure to provide as much information as possible, including script output so
+    we can collect information about your configuration. This helps reduce the time required to
+    reproduce your issue.
+  * Check your issue regularly, as we may require additional information to successfully reproduce the
+    issue.
+* You may also open an issue to ask questions to the maintainers about whether a proposed change meets the acceptance criteria, or to discuss an idea pertaining to the library. 
+
+<!-- [PART1] Instructions:
+Above is fairly standard.  Please add any special instructions (if any) your team may have regarding issues -->
+
+
+## Acceptance Criteria ## 
+
+<!-- [PART2] Instructions:
+Include details of your library such as:
+   1. The overarching goals of a library
+   2. The sample features that fulfils those goals
+   3. Any performance criteria needed for those features.
+
+The specification of performance criteria shouldn't include exact numeric values i.e x% perf gains.
+However, it should include some language suggesting the need of notable performance gain to avoid
+code duplication and code maintenance if no gain is found. Contributors should be guided about 
+any performance benchmarking mechanism including baseline comparison. 
+
+Additionally, to encourage contributors, we should point out that the "Issue" tab, as defined below, 
+could be leveraged by them to discuss with the community members, features they would like to 
+bring in or the purported perf gains it would bring, before raising pull requests.
```

Comment:
This sentence is difficult to follow. Can we break it up?

Reply:
Yeah, reworded this whole section from Tejash... hopefully I got across his points.